.github/dependabot.yml

```diff
@@ -59,13 +59,19 @@ updates:
       - nuget-org
     schedule:
       interval: "daily"
+    ignore:
+      # Starting with version 16.10.0, MSBuild requires .NET 5.0+
+      - dependency-name: "Microsoft.Build"
```

Comment:
Why are we ignoring this package?

Reply:
We are using MSBuild 16.9.0, but starting with the next release (16.10.0), the project will require .NET 5.0 or greater. Since PA Client targets netcoreapp3.1, we were getting a package restoration issue, and the package was being resolved for .NET Framework instead. This led to hanging in our unit tests.

The target framework versions can be compared on Nuget:
* [Version 16.10.0](https://www.nuget.org/packages/Microsoft.Build/)
* [Version 16.9.0](https://www.nuget.org/packages/Microsoft.Build/16.9.0)

At some point we'll probably need to upgrade to .NET5 or .NET6 at which point we can stop ignoring this package.

Reply:
Sounds good. thanks for the explanation